tensorboard/webapp/metrics/views/card_renderer/utils.ts

```diff
@@ -12,22 +12,25 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
+import {ExperimentAlias} from '../../../experiments/types';
 import {Run} from '../../../runs/store/runs_types';
 import {LinkedTime} from '../../types';
 import {PartialSeries, PartitionedSeries} from './scalar_card_types';
 
 export function getDisplayNameForRun(
   runId: string,
   run: Run | null,
-  experimentName: string | null | undefined
+  experimentAlias: ExperimentAlias | null | undefined
 ): string {
-  if (!run && !experimentName) {
+  if (!run && !experimentAlias) {
     return runId;
   }
 
-  const displayName = [experimentName, run ? run.name : '...']
-    .filter(Boolean)
-    .join('/');
+  let displayName = [run ? run.name : '...'].filter(Boolean).join('/');
```

Comment:
This array only has one item so this feels incorrect. Just do `displayName = run ? run.name : '...'` or `displayName = run?.name ?? '...'` (which functionally is different but run.name should not be empty as it can break other UI so default showing "..." feels good.)

Reply:
done.